### PR TITLE
First draft of accounts_statement schema

### PR DESCRIPTION
**Pull request by CountCulture**

### CODE REVIEW

**Comment by CountCulture** [comment]
More attributes to be added, but this is draft structure. Matches GbXbrlParser one, so we can switch that to turbot bot when finalised.
NB sample_date is duplicate of accounts date, which is ugly. However, I think we should explicitly capture the accounts date.
Comments welcome @sebbacon

File: schemas/accounts-statement-schema.json [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+      "description": "Should be represented as two-letter ISO-639 code",
+      "type": "string",
+      "minLength": 2,
+      "maxLength": 2
+    },
+    "source_location": {
+      "type": "string",
+      "description": "If the accounts filing has been put on sources.opencorporates.com the path to the filing"
+    },
+    "confidence": {
```
Comment (sebbacon) [inline schemas/accounts-statement-schema.json]: Got this twice

**Comment by sebbacon** [comment]
This comment from the other PR was intended for here (but it does apply to both):

> I think we *either* need to specify this is for annual accounts only, *or* also support an accounting period (either from `date_previous_accounts_made_up_to` or `annual`, `quarterly`, etc.

**Comment by CountCulture** [comment]
I'm going to add accounts type initially, to help distinguish between annual accounts and quarterlies (important for SEC filings). This is different from the dates, however, as annual accounts (e.g. those filed at Companies House) can be longer or shorter than 1 year.

For the start date and end dates of the accounts period I wonder if start_date and end_dates should perhaps be used  instead. In addition, in XBRL accounts (which I think are a pretty good representation of the underlying accounts) periods are for each attribute, not for the whole accounts

**Comment by sebbacon** [comment]
OK, seems sensible

**Pull request merged**